cpp/tests/groupby/sort/ignore_null_keys/legacy/single_column_var.cu

```diff
@@ -0,0 +1,234 @@
+/*
+ * Copyright (c) 2019, NVIDIA CORPORATION.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#include <tests/utilities/legacy/cudf_test_fixtures.h>
+#include <cudf/legacy/groupby.hpp>
+#include <cudf/legacy/table.hpp>
+#include <tests/utilities/legacy/column_wrapper.cuh>
+#include <tests/utilities/legacy/compare_column_wrappers.cuh>
+#include <cudf/utilities/legacy/type_dispatcher.hpp>
+#include <tests/groupby/sort/legacy/single_column_groupby_test.cuh>
+#include <tests/groupby/common/legacy/type_info.hpp>
+
+#include <gmock/gmock.h>
+#include <gtest/gtest.h>
+
+#include <random>
+
+static constexpr cudf::groupby::operators op{
+    cudf::groupby::operators::VARIANCE};
+
+template <typename KV>
+struct SingleColumnVariance : public GdfTest {
+  using KeyType = typename KV::Key;
+  using ValueType = typename KV::Value;
+};
+
+template <typename T>
+using column_wrapper = cudf::test::column_wrapper<T>;
+
+template <typename K, typename V>
+struct KV {
+  using Key = K;
+  using Value = V;
+};
+
+using TestingTypes =
+    ::testing::Types< KV<int32_t, int32_t> >;
```

Comment:
Only int32 types tested? I would think you would want to test float types for variance...